lib/screens/survey/survey_screen.dart

```diff
@@ -0,0 +1,114 @@
+import 'package:flutter/material.dart';
+import 'package:survey_flutter/gen/assets.gen.dart';
+import 'package:survey_flutter/model/survey_model.dart';
+import 'package:survey_flutter/theme/app_constants.dart';
+import 'package:survey_flutter/theme/primary_button_style.dart';
+import 'package:survey_flutter/utils/build_context_ext.dart';
+
+const routePathSurveyScreen = '/survey';
```

Comment:
It would be better if we define a `route_path.dart` which only contains the route path of the app